site/_js/web-components/search-box.js

```diff
@@ -29,6 +29,15 @@ import {activateSearch, deactivateSearch} from '../actions/search';
 const client = algoliasearch('0PPZV3EY55', 'dc0d3a2d53885be29eacc351026dcdcf');
 const index = client.initIndex('prod_developer_chrome');
 
+const blockedQueries = [
+  /add? bloc?k/,
+  /d(a|o)w?nl?o?/,
+  /^p(a|e)r?r(a|e)?/,
+  /automate be/,
+  /vpn/,
+  /xxx/,
+];
```

Comment:
I'd make a blacklist in a separate yaml file, easily editable without touching the code.

Reply:
Held back from doing so as it's client-side code and our Roll Up config doesn't support transforming YAML files, but also because after thinking about it I think having it in one place is more intuitive as we don't have a place for client-side configs yet.